addons/stock_account/models/product.py

```diff
@@ -162,7 +291,7 @@ def action_revaluation(self):
     # -------------------------------------------------------------------------
     # SVL creation helpers
     # -------------------------------------------------------------------------
-    def _prepare_in_svl_vals(self, quantity, unit_cost):
+    def _prepare_in_svl_vals(self, quantity, unit_cost, lot_id=False):
```

Comment:
lot if recordset.
lot_id if id
change the method or the arg passed to it (seems we don't need the record)